src/OpenSage.Game.Tests/Gui/Apt/ActionScript/InstructionParsingTest.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System.IO;
+using static System.Text.Encoding;
+using OpenSage.Gui.Apt;
+using OpenSage.Gui.Apt.ActionScript;
+using OpenSage.Gui.Apt.ActionScript.Opcodes;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OpenSage.Tests.Gui.Apt.ActionScript
+{
+    static class Utilities
+    {
+        public static void Align(this BinaryWriter writer)
+        {
+            while (writer.BaseStream.Position % 4 != 0)
+            {
+                writer.Write((byte)0);
+            }
+        }
+
+        public static void WriteSimpleInstruction(this BinaryWriter writer, InstructionType type, int? argument = null)
+        {
+            writer.Write((byte)type);
+            if (argument.HasValue)
+            {
+                if (InstructionAlignment.IsAligned(type))
+                {
+                    Align(writer);
+                }
+                writer.Write(argument.Value);
+            }
+        }
+
+        public static int WriteNullTerminatedString(this BinaryWriter writer, string value)
+        {
+            writer.Align();
+            var position = (int)writer.BaseStream.Position;
+            writer.Write(UTF8.GetBytes(value + '\0'));
+            return position;
+        }
+    }
+
+    public class InstructionParsingTest
+    {
+        private readonly ITestOutputHelper _output;
+
+        public InstructionParsingTest(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        
```

Comment:
Could you just have one empty line here?

Reply:
All right, will fix them soon